src/main/java/codeu/model/data/Activity.java

```diff
@@ -0,0 +1,60 @@
+package codeu.model.data;
+
+/**
+ * Class representing an activity, which can be thought of as any action
+ * within the chat room. Activities are triggered by a User joining a
+ * conversation, starting a conversation or replying to a message.
+ */
+public class Activity {
```

Comment:
I would recommend creating a constructor that creates a fully initialized `Activity` rather than making them settable by consumers of it.

Then you can make `creationTime` and `output` `private final String`s